playbooks/support/README.md

```diff
@@ -19,111 +50,167 @@ which is determined at least a month in advance (giving people ample time to mak
 
 During work hours, on-call engineers are responsible for responding to issues in
 [the #incidents slack channel](https://artsy.slack.com/messages/C9RK0BLEP/) 🔒 in a timely fashion. Outside of work
 hours, engineers are _only_ responsible for downtime issues.
 
-The two engineers who are on-call are free to split the responsibility for monitoring outside of work hours, but we
-suggest either doing every other day or first half/second half.
-
 ## On-Call Responsibilities
 
 While on-call, you are accountable for investigating and fixing timely issues, escalating to additional
 point-people and/or routing to team-specific backlogs where appropriate.
 
-Your top priority during an on-call period is to address critical issues. Use your judgment about participating in
-ongoing sprint and team activities.
+Your top priority during an on-call period is to ensure the resolution of critical issues. Use your judgment about
+participating in ongoing sprint and team activities.
+
+While we would _like_ to have 24/7 on-call coverage, we recognize that we don't yet have a good solution for
+notifying engineers during non-work hours. Do your best to keep an eye out for issues, and lean on those in more
+favorable timezones where appropriate.
+
+## Responding to an incident
+
+### Step 1: Alert
+
+Incidents can be raised by automated or manual means. While on-call, you are responsible for being available to
+respond to these issues.
+
+- Monitor the [#incidents slack channel](https://artsy.slack.com/messages/C9RK0BLEP/) 🔒 for critical issues. This
+  is where urgent issues are raised, either manually or via automated alerts.
+- Monitor the [#alerts](https://artsy.slack.com/messages/C0HP61PUJ/) slack channel _during business/waking hours_.
+  These automated system alerts _may_ be the first sign of a problem but don't _necessarily_ warrant treatment as
+  immediate incidents. E.g., load-balancers may route around failing EC2 instances or auto-scaling may bring up
+  capacity in response to load.
+- If the issue is obviously a non-incident, take the time to educate the reporter on the desired process (see the
+  [examples](#examples) below). You may point them to the
+  [Guide to Reporting Bugs](https://www.notion.so/artsy/Guide-to-reporting-bugs-cc25e1ff41194228b476c4963c646817)
+  doc in Notion which describes this in a general way.
+
+### Step 2: Raise
+
+1. When an incident is raised in the channel or identified in another way, the on-call team first **tracks the
+   incident** by creating a ticket in [Jira Ops](https://artsyproduct.atlassian.net/projects/INCIDENT/incidents),
+   based on the information they know at that time. Some workflow steps:
+
+- In the description on the ticket, link to the slack thread in #incidents.
+- Take your best guess at the other information, it can always be edited later.
+
+2. Using the slack integration on the Jira Ops ticket, create a new slack channel to discuss resolution.
```

Comment:
I'm not sure if this should be in another RFC or not, but I'm strongly of the opinion that defaulting to synchronous conversation is helpful in any incident response and facilitates quick resolution of an incident. It's also recommended in [Jira Ops playbook](https://www.atlassian.com/software/jira/ops/handbook/responding-to-an-incident#faq-d5ae6e39-9ddf-4a6b-b279-830b829ab146).

As such, I propose adding another bullet here with:

> 3. Start a Zoom in the incident specific channel using the `/zoom` slack command

Reply:
Since there's still a bit to be figured out here, maybe it can be a separate PR?